### ISSUE TITLE
Error when showing job results

### ISSUE THREAD

**Issue by jdries** [Open-EO/openeo-vue-components#58]
Trying to show batch job results for the Terrascope backend fails with an error, see below.
I guess our STAC metadata could be incorrect, but can't figure out from the error what the problem is.

![image](https://user-images.githubusercontent.com/5937096/126622953-39e7c19a-c0cd-4ae4-94d6-a495f9085753.png)

**Comment by jdries** [comment]
Not sure why yet, but the 'data' object in the response indeed seems to be 'null'.

**Comment by m-mohr** [comment]
Which client are you using? Jupyter? Web Editor?

**Comment by jdries** [comment]
web editor!

**Comment by m-mohr** [comment]
Yeah, just realized from the last line in the stack trace, will check. What's the state of the job?

**Comment by m-mohr** [comment]
Is that editor.openeo.org or editor.openeo.cloud and are you connected to VITO directly or to the aggregator?
